Example/Swift Example/UserQueryViewController.swift

```diff
@@ -19,10 +19,57 @@
 
 import Foundation
 import SKYKitChat
+import Kingfisher
 
 class UserQueryViewController: SKYChatParticipantListViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         self.queryMethod = .ByName
+        self.dataSource = self
+    }
+}
+
+extension UserQueryViewController: SKYChatParticipantListViewControllerDataSource {
+    func listViewController(_ controller: SKYChatParticipantListViewController,
+                            avatarImageForParticipant participant: SKYRecord,
+                            atIndexPath indexPath: IndexPath) -> UIImage? {
+
+        var name = ""
+        if let participantName = participant.object(forKey: "name") as? String {
+            name = participantName
+        }
+
+        let placeholderImage =  UIImage.avatarImage(forInitialsOfName: name)
+
+        // demonstrate how to set avatar asynchronizely
```

Comment:
s/asynchronizely/asynchronously/g

In examples, try write comments that is intended for users of your SDK, not from the view of the SDK developer.

In this instance, “demonstrate how to set avatar asynchronously” explain to the developer why this snippet of code is here. But explaining to a user of the SDK involves different word choice.

Try: If you want to use an remote image as the avatar image, you need to fetch the image asynchronously and return a placeholder such that the UI can display something right away. When the image is fetched, we call `update` function to update the avatar image in the `UITableViewCell`.



This explained why the user of the SDK wants to add this snippet to their code and how this snippet works.